src/MinimumNodeService.cpp

```diff
@@ -212,18 +212,25 @@ Processed MinimumNodeService::handleMessage(unsigned int opc, CANFrame *msg)
 
       if (nn == module_config->nodeNum)
       {
-        byte paran = msg->data[3];
+        if (msg->len < 4)
+        {
+          controller->sendGRSP(OPC_RQNPN, getServiceID(), CMDERR_INV_CMD);
```

Comment:
Should this condition also send a CMDERR?

Reply:
Sorry, I see now that this is a VLCB specific check. So no CMDERR.

Reply:
It doesn't seem to be specified in CBUS and Duncan didn't in his code, so I think not.

Reply:
Ian doesn't in his code, either